CHANGELOG.asc

```diff
@@ -6,4 +6,35 @@ Release Notes
 Version 0.1.0 (TBD)
 ~~~~~~~~~~~~~~~~~~~
 
-Changelog to be added once released.
+[source, m2]
+<dependency>
+   <groupId>org.janusgraph</groupId>
+   <artifactId>janusgraph-core</artifactId>
+   <version>0.1.0</version>
+</dependency>
+
+*Tested Compatibility:*
+
+* Cassandra 2.1.9
+* HBase 0.98.8-hadoop2, 1.0.3, 1.1.8, 1.2.4
+* BerkeleyJE 7.3.7
```

Comment:
Changed to "live" for consistency with #176. Depending on the response to [this question](https://github.com/JanusGraph/janusgraph/pull/176#issuecomment-290964057) this might need to be changed again though.